src/simperium/channel.js

```diff
@@ -237,6 +238,9 @@ internal.handleChangeError = function( err, change, acknowledged ) {
 				this.localQueue.dequeueChangesFor( change.id );
 			}
 
+			break;
+		case CODE_DUPLICATE_CHANGE:
+			internal.updateAcknowledged.call( this, acknowledged );
```

Comment:
I left this as a separate `case` since it carries a different semantic meaning than an empty change